components/brave_rewards/browser/rewards_service.h

```diff
@@ -65,8 +65,8 @@ using GetReconcileStampCallback = base::Callback<void(uint64_t)>;
 using IsWalletCreatedCallback = base::Callback<void(bool)>;
 using GetPendingContributionsTotalCallback = base::Callback<void(double)>;
 using GetRewardsMainEnabledCallback = base::Callback<void(bool)>;
-using GetTransactionHistoryForThisCycleCallback =
-    base::Callback<void(int, double)>;
+using GetTransactionHistoryCallback =
+    base::Callback<void(double, uint64_t, uint64_t)>;
```

Comment:
can you change this to `OnceCallback`? Every new/modified method we would like to migrate to once